### PR TITLE
Added Lightlink Mainnet and LightLink Goerli Testnet

### PR DESCRIPTION
**Pull request by vanshwassan**
Added new chains:
- [LightLink Phoenix Mainnet](https://phoenix.lightlink.io/)
-  [LightLink Pegasus Testnet](https://pegasus.lightlink.io/)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@api3/chains",
   "license": "MIT",
-  "version": "4.1.1",
+  "version": "4.1.2",
```
Comment (andreogle) [inline package.json:4]: Would you mind setting this back to 4.1.1 please? There's [another PR](https://github.com/api3dao/chains/pull/138) I'd like to include and then I can release v4.1.2 (makes it easier to find things in the commit history)

**Review by andreogle — APPROVED** [review]
Looks good - I see it's still a draft though so let me know when you're happy with it being merged 👍🏻

File: package.json (hunk 1 above)
Comment (bbenligiray) [reply package.json:4]: I think adding providers require a minor release and I published that because it was blocking @bdrhn9

**Comment by bbenligiray** [comment]
I'm wondering if we should implement some kind of tiers between chains in this package (ones that are supported by API3 market and others) because I don't want us to be bogged down by boba-avalanche tier stuff caused by hundred of chain integrations such as this one

File: package.json (hunk 1 above)
Comment (andreogle) [reply package.json:4]: > I think adding providers require a minor release

Yeah that's probably more correct

**Comment by bbenligiray** [comment]
I guess just a flag that disables provider CI for certain chains for the moment. When we have hundred of chains (where we don't provide any services on half of them) we may need to do more.

**Comment by vanshwassan** [comment]
Closing this PR for now. Will wait on either having a better way to accommodate other chains/ flags to disable CI for certain chains.

**Comment by vanshwassan** [comment]
Added the `skipProviderCheck` flag and reopened the PR. For reference, check [this](https://github.com/api3dao/airnode/pull/1924#issuecomment-1887437755)

File: chains/lightlink-goerli-testnet.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "id": "1891",
+  "name": "LightLink Goerli Testnet",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://replicator.pegasus.lightlink.io/rpc/v1"
+    }
+  ],
+  "skipProviderCheck": false,
```
Comment (andreogle) [inline chains/lightlink-goerli-testnet.json:22]: I think this is the wrong way around 😬 

```suggestion
  "skipProviderCheck": true,
```

File: chains/lightlink.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "id": "1890",
+  "name": "LightLink",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://replicator.phoenix.lightlink.io/rpc/v1"
+    }
+  ],
+  "skipProviderCheck": false,
```
Comment (andreogle) [inline chains/lightlink.json:22]:
```suggestion
  "skipProviderCheck": true,
```

**Review by andreogle — CHANGES_REQUESTED** [review]

File: chains/lightlink-goerli-testnet.json (hunk 2 above)
Comment (vanshwassan) [reply chains/lightlink-goerli-testnet.json:22]: oh, my bad. fixing it

**Pull request merged**